hathor/p2p/peer.py

```diff
@@ -122,6 +122,20 @@ class PeerInfo:
     flags: set[str] = field(default_factory=set)
     _settings: HathorSettings = field(default_factory=get_global_settings, repr=False)
 
+    @property
+    def ipv4_entrypoints(self) -> list[Entrypoint]:
+        return list(filter(lambda e: not e.is_ipv6(), self.entrypoints))
+
+    @property
+    def ipv6_entrypoints(self) -> list[Entrypoint]:
```

Comment:
I'd rather rename it to `get_ipv6_only_entrypoints()`.

Reply:
Will do

Reply:
Done

Reply:
Thanks!